### PR TITLE
Technical Lead Role Description

### PR DESCRIPTION
**Pull request by imnmo**
Detailed view:

HackMD :
https://hackmd.io/Gjudrpj0RhSlPAGOhi58fg

### CODE REVIEW

**Comment by imnmo** [comment]
cc: @kaslin @chris-short

**Comment by jberkus** [comment]
/assign @kaslin 

Kaslin, please review.

**Comment by kaslin** [comment]
Update name to Comms Tech Lead or something like that to differentiate from other roles in ContribEx.

File: communication/marketing-team/role-handbooks/Tech-Lead.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Tech lead Role Handbook
```
Comment (reylejano) [inline communication/marketing-team/role-handbooks/Tech-Lead.md:1]: nit: looks like title case is used for headers, the following suggestion keeps the pattern
```suggestion
# Tech Lead Role Handbook
```

File: communication/marketing-team/role-handbooks/Tech-Lead.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+- Take ownership of an issue and submit a PR for repositories like (especially if it's quick or relates to work you're already doing) Contributor-site, tweets, and, slack bot
+- Have an understanding of what an issue is about and able to triage them, or know who to contact to determine the required information
+- Coordinate work to fix any issues in automation tools to ensure valid tweet requests are fulfilled
+- Train new leads
+
+Team members will be expected to keep an eye out for news or issues related to slack bot or Github automation. Common sources of such announcements are the k-dev mailing list and the #contributor-comms channel on Slack.
+
+## Associates Overview (Shadow)
+
+Team members who wish to become technical leads shadow should express this desire to the current lead.
```
Comment (reylejano) [inline communication/marketing-team/role-handbooks/Tech-Lead.md:42]: nit
```suggestion
Team members who wish to become technical lead shadow should express this desire to the current lead.
```

File: communication/marketing-team/role-handbooks/Tech-Lead.md (hunk 1 above)
Comment (imnmo) [reply communication/marketing-team/role-handbooks/Tech-Lead.md:1]: done

**Pull request merged**